### DISCUSSION TITLE
Calling mask commands from another mask command

### DISCUSSION THREAD

**Discussion by whavey** [jacobdeichert/mask#105] [Q&A] [answered]
How would I go about referencing previously defined mask commands in one? Lets say I have a build and a run mask command defined and I want to make one called deploy without reusing the code from both in the new one.

**Comment by jacobdeichert** [comment, marked as answer]
Hey! Can you provide an example of what you mean?

It sounds like [this section](https://github.com/jacobdeichert/mask#running-mask-from-within-a-script) of the README might help?

**Reply by whavey** [reply]
Yep that was it sorry I missed that I think I saw the heading title and thought it was talking about something else. Thanks!
